README.md

```diff
@@ -1,207 +1,47 @@
-# Template for setting up reproducible projects
-<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
-[![All Contributors](https://img.shields.io/badge/all_contributors-6-orange.svg?style=flat-square)](#contributors-)
-<!-- ALL-CONTRIBUTORS-BADGE:END -->
+# Open Seeds Self-organised 
 
-*This is a template for setting up a research or data science project online, such as on GitHub.*
+Welcome! This is the [Self-Organised](https://we-are-ols.org/openseeds/) (OLS- Self-Organised) program repository. 
+This repository will be used to provide resources for individuals and communities who want to run the OLS Open Seeds program but couldn't get a chance to participate during the earlier cohorts. Also, the resources here could be useful to folks who want to run the program self-space and independently
 
-## About this Repository
 
-This repository operationalises The Turing Way recommendations for structuring project repositories to support reproducibility in data science projects.
-This repository includes files and directories recommended for enabling reproducibility and collaboration in a project, as well as sharing of research objects.
+### Participation Guidelines
 
-This repository follows the recommendations and guidance provided in *[The Turing Way](https://the-turing-way.netlify.app/welcome)* handbook to data science.
-When reusing this repository, please update the information on your README page with information about your project.
+Open Life Science participants adhere to a [code of conduct](CODE_OF_CONDUCT.md). 
+By participating, you are expected to uphold this code. Please report unacceptable behaviour to [team@we-are-ols.org](mailto:team@we-are-ols.org).
 
-### Using this Repository
+### Graduation guide
 
-![screenshot with arrows and instructions describing how to use this repository template to create a project repo. The description provided in the title.](./assets/use-this-repo.png)
+To learn about graduation criteria and process, please visit this [Graduation Guide](https://we-are-ols.org/openseeds/cohort-procedures-and-templates/graduation-guide.html).
```

Comment:
Graduation guide doesn't quite fit here. 

We need to provide info about what is required to run or follow Open Seeds. This repo is for the organisers primarily. We can use a lightweight version of this document as provided for Ally Skills facilitators - see the table of content to see what we might need to communicate: https://docs.google.com/document/d/1NTnVaNU4hgLHLKHoEFEyCI3Zg2BdWZjyYeKOKAGiHXc/edit?tab=t.0